src/pybind/mgr/dashboard/frontend/src/locale/messages.zh-CN.xlf

```diff
@@ -174,7 +174,7 @@
       </trans-unit>
       <trans-unit id="3859bdca1ae89e611e214c205fccf0147b4d0f6b" datatype="html">
         <source>The value is not a valid IP address.</source>
-        <target>The value is not a valid IP address.</target>
+        <target>不是有效的IP地址。</target>
```

Comment:
I just checked and this is not just happening with chinese language but with all other languages, so there's something that's not happening right.